.github/pull_request_template.md

```diff
@@ -6,6 +6,7 @@
 ### Manual PR Checklist
 
 - [ ] CHANGELOG entry added
+- [ ] I ran the build command after making changes to the schema
```

Comment:
this can be automated in CI.. I would remove this from the template, and create a ticket in JIRA to automate the check.. wdyt @kgrhartlage ?

Reply:
Correct.. we have something similar for the [middleware](https://github.com/labforward/laboperator-middleware/pull/23/commits/940cf52658fb94cdb40d3fe85a9e9a060f5403e3#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R53-R62)..